src/python/grpcio/grpc/experimental/aio/_channel.py

```diff
@@ -167,10 +169,17 @@ def __call__(self,
 
         deadline = _timeout_to_deadline(timeout)
 
-        call = StreamUnaryCall(request_iterator, deadline, metadata,
-                               credentials, wait_for_ready, self._channel,
-                               self._method, self._request_serializer,
-                               self._response_deserializer, self._loop)
+        if not self._interceptors:
+            call = StreamUnaryCall(request_iterator, deadline, metadata,
+                                   credentials, wait_for_ready, self._channel,
+                                   self._method, self._request_serializer,
+                                   self._response_deserializer, self._loop)
```

Comment:
optional: At some point, we might want to reduce the number of arguments (this one has 10) with some data structure.

Reply:
yes, I promise that will be no more arguments!